Cesium.Runtime/StdLibFunctions.cs

```diff
@@ -5,6 +5,9 @@ namespace Cesium.Runtime;
 /// </summary>
 public unsafe static class StdLibFunctions
 {
+    public const int RAND_MAX = 0x7fff;
```

Comment:
In GCC, this is `0x7FFFFFFF`. I think that would be better, since it is more in line with `Random.Next()`.